modules/microshift-otel-config.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+//  microshift_running_apps/microshift-observability-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-otel-config_{context}"]
+= Configuring {microshift-short} Observability
+
+You can configure {microshift-short} Observability after it is installed to identify a valid endpoint.
+
+[IMPORTANT]
+====
+You must specify an OpenTelemetry Protocol (OTLP)-compatible endpoint for each configuration, such as Prometheus, before starting {microshift-short}. If an endpoint is not specified, {microshift-short} Observability will not start.
```

Comment:
OTLP compatible exporter is not mandatory, see all available exporters: https://opentelemetry.io/docs/collector/configuration/#exporters

Reply:
@pmtk @copejon pinging you here in case I missed something from requisites.
All our example configs (small, medium and large) only use `otlp` as exporter, which is fine. But, as far as I understand, it's not mandatory to use `otlp` exporter, you can use any other exporter.

Reply:
Is it about the exporter? I think this is about the backend.
Although "such as Prometheus" might be misleading.

Reply:
I agree with @pmtk in this comment.